src/app/shared/deviceSniffingHelper.ts

```diff
@@ -1,35 +1,54 @@
 import { UAParser } from "ua-parser-js";
 import { Request } from "express";
-import { PHONE_TYPES, OS_TYPES } from "../../constants/device-constants";
+import {
+  PHONE_TYPES,
+  OS_TYPES,
+  MINIMUM_IOS_VERSION,
+} from "../../constants/device-constants";
 import { APP_TRIAGE_EVENTS } from "../../constants/events";
 
+export interface OsType {
+  name: string;
+  version?: number;
+}
+
 // The AppTriage event is special in that we want to send a more specialised version if we can detect the current
 // device type as being an Android phone or iPhone.
 export const detectAppTriageEvent = (req: Request): string => {
   const detectedPhone = sniffPhoneType(req);
 
-  switch (detectedPhone) {
+  switch (detectedPhone?.name) {
     case PHONE_TYPES.ANDROID:
-      return APP_TRIAGE_EVENTS.APP_TRIAGE_ANDROID;
+      return APP_TRIAGE_EVENTS.MOBILE_DOWNLOAD_ANDROID;
     case PHONE_TYPES.IPHONE:
-      return APP_TRIAGE_EVENTS.APP_TRIAGE_IPHONE;
+      if (
+        detectedPhone?.version &&
+        detectedPhone.version < MINIMUM_IOS_VERSION
+      ) {
+        return APP_TRIAGE_EVENTS.APP_TRIAGE_SMARTPHONE;
+      }
+      return APP_TRIAGE_EVENTS.MOBILE_DOWNLOAD_IPHONE;
     default:
       return APP_TRIAGE_EVENTS.APP_TRIAGE;
```

Comment:
Hmm I'm wondering, if we don't detect a version at all (ie. it's empty / null) we should probably treat that as device sniffing fails and go to the smartphone triage page? (even for Android)